src/node_binding.cc

```diff
@@ -52,6 +52,7 @@
   V(mksnapshot)                                                                \
   V(options)                                                                   \
   V(os)                                                                        \
+  V(path)                                                                      \
```

Comment:
Not a strong opinion, but maybe we should name the binding and file `glob` instead of `path`?

Reply:
@cjihrig I'm planning on moving a couple of functions to C++ (actually done the same exact change), so this change is beneficial for me too.